x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/DocsV3SupportTests.java

```diff
@@ -418,7 +404,7 @@ public static class TestClass extends Function {
                 @Example(description = """
                     To count the number of times an expression returns `TRUE` use a
                     <<esql-where>> command
-                    to remove rows that shouldn’t be included""", file = "stats", tag = "count-where"),
+                    to remove rows that shouldn't be included""", file = "stats", tag = "count-where"),
```

Comment:
This is wrong. We took the decision in March to put in the correct typesetting apostrophe because MD does not render it correctly, while asciidoc does. This appears to reverse that decision.

Reply:
unintended side effect, and unfortunately a lot less important than being able to GA the search functions for 9.1